cmd/installer/kotscli/kotscli.go

```diff
@@ -79,8 +84,16 @@ func Install(opts InstallOptions) error {
 		installArgs = append(installArgs, "--airgap-bundle", opts.AirgapBundle)
 		maskfn = MaskKotsOutputForAirgap()
 	}
+
 	if opts.ConfigValuesFile != "" {
 		installArgs = append(installArgs, "--config-values", opts.ConfigValuesFile)
+	} else if opts.ConfigValues != nil {
+		configValuesFile, err := createConfigValuesFile(opts.ConfigValues, opts.RuntimeConfig)
```

Comment:
i think the caller should simply create the file and pass it via `ConfigValuesFile` instead of having the kots cli do this step. that keeps the kotscli package a light wrapper around the kots cli and easy to take out in the future.

Reply:
actually, a better suggestion maybe is to pass config values as a map and then this package converts it to a `kind: ConfigValues` file. that way, the ConfigValues logic is only coupled to the KOTS package.